CSLabs.Api/Controllers/ModuleController.cs

```diff
@@ -49,13 +66,22 @@ public async Task<IActionResult> Get(int id)
         
         [HttpGet("code/{code}")]
         [AllowAnonymous]
-        public async Task<IActionResult> Get(string code)
+        public async Task<IActionResult> GetCode(string code)
         {
             var module = await this.DatabaseContext.Modules.FirstAsync(m => m.SpecialCode == code);
             await module.SetUserModuleIdIfExists(DatabaseContext, GetUser());
             return Ok(module);
         }
-        
+
+        [HttpGet("{id}/tags")]
+        public IActionResult GetTags(int id)
+        {
+            var moduleTags = this.DatabaseContext.Modules
+                .Where(m => m.Id == id)
+                .SelectMany(m => m.ModuleTags).ToList();
+            var tags = moduleTags.Select(mt => this.DatabaseContext.Tags.Single(t => t.Id == mt.TagId)).ToList();
+            return Ok(tags);
+        }
```

Comment:
This method can be written into a one liner.

```suggestion
        [HttpGet("{id}/tags")]
        public async Task<IActionResult> GetTags(int id) =>
            Ok(await DatabaseContext.Tags.Where(t => t.ModuleTags.Any(mt => mt.ModuleId == id)).ToListAsync());
```

Reply:
This method may not be needed if the tags are returned in the get module request.